wordpress-mobile/WordPress-Android
# Publish notification dialog

This PR introduces logic for enabling/disabling/hiding of the "Notification" field in the publish settings. Once the field is enabled, there is a dialog that allows the user to select the notification time (`OFF`, `ONE_HOUR_BEFORE`, `TEN_MINUTES_BEFORE`, `WHEN_PUBLISHED`) which will be used to schedule notifications in future PRs. 

To test:
* Go to Publish settings
* Set the date into the past
* Notice that the "Notification" field is hidden
* Set the date to "Immediately/now"
* Notice that the "Notification" field is disabled
* Set the date to future
* Notice that the "Notification" field is enabled
* Click on the "Notification" field
* Select an item from the dialog
* Click on "OK"
* Notice that the selected item is now visible in the "Notification" field

![Screenshot_1563293061](https://user-images.githubusercontent.com/1079756/61310230-2e2bec00-a7f4-11e9-8d11-3cf92cda512c.png)

Update release notes:
